src/cmds/sys/env.c

```diff
@@ -68,8 +68,7 @@ int main(int argc, char *argv[]) {
 	/* difference from posix: execv return 0 and does not set errno on errors */
 	if (optind < argc) {
 		execv(argv[optind], argv+optind);
-		printf("env: execv: No such command '%s'\n", argv[optind]);
```

Comment:
I think It's better to keep printf but the message change to "env: execv: command '%s' failed with error code %d"

Reply:
Okay, will make this change, but that'll mean I'll be printing errno(since ecode won't be returned from execv, as POSIX requires execv to return -1 on errors). 
So if I'm printing errno, isn't it better to use perror()?

Reply:
Ok, just remove.
Thanks